packages/blocks/src/database-block/table/components/column-type/select/select-cell-editing.ts

```diff
@@ -97,6 +97,12 @@ const styles = css`
   .select-selected > .close-icon {
     display: flex;
     align-items: center;
+    fill: var(--affine-icon-color);
+  }
+
+  .select-selected > .close-icon:hover {
+    cursor: pointer;
+    background: var(--affine-hover-color);
```

Comment:
The hover `background` is not needed here, this is also by design.

Reply:
Thanks again. By the way, should I keep the cursor: pointer or not?

Reply:
> Thanks again. By the way, should I keep the cursor: pointer or not?

Yes. Generally speaking, the clickable area should be indicated by the mouse style(`cursor: pointer`).

Reply:
Thanks